cargo-generate.toml

```diff
@@ -24,15 +24,15 @@ prompt = "Do you want to use Nix & Nix Flakes?"
 default = false
 
 [conditional.'!flakes']
-ignore = [ "flake.nix", "flake.lock", ".envrc" ]
+ignore = ["flake.nix", "flake.lock", ".envrc"]
 
 [placeholders.docker]
 type = "bool"
 prompt = "Do you want to generate a Dockerfile for your gadget?"
 default = true
 
 [conditional.'!docker']
-ignore = [ "Dockerfile", "docker/" ]
+ignore = ["Dockerfile", "docker/"]
 
 [conditional.'docker'.placeholders]
-base-image = { type = "string", prompt = "What base image should be used?", default = "rustlang/rust:nightly" }
\ No newline at end of file
+base-image = { type = "string", prompt = "What base image should be used?", default = "rustlang/rust:nightly-slim" }
```

Comment:
Did you test building and running the image?

Reply:
Nope. Is there an issue with this image? Though slim one would make the result image lighter.

Reply:
I haven't tried it yet, just want to make sure it works with cargo chef. I'll test it in a minute.